packages/mgt-chat/src/components/ChatListHeader/ChatListHeader.tsx

```diff
@@ -39,27 +50,34 @@ export const ChatListHeader = (
   props: MgtTemplateProps &
     IChatListMenuItemsProps & {
       buttonItems?: ChatListButtonItem[];
+      bannerMessage?: string;
+      bannerClassName?: string;
```

Comment:
What's the purpose of bannerClassName being a prop?

Reply:
it was the desire to provide some functionality earlier to change the UX, but after trying it out, it seemed to be more complicated matter, so I stopped working on it, but forgot to remove it.